winforms/src/toga_winforms/widgets/base.py

```diff
@@ -4,7 +4,20 @@
 from toga_winforms.libs import Color, Point, Size, SystemColors
 
 
-class Widget:
+class Scalable:
```

Comment:
Any particular reason for making a subclass for this? It's only ever subclassed by Widget, and it's not used as a basis for isolated testing; I'm not sure I see the benefit of a separate base class for this.

Reply:
It's also subclassed by Window, and used in the position and size methods. This was required by the OptionContainer tests, because they rely on the window being 640x480 CSS pixels.